src/OpenSage.Game/Gui/Apt/AptSystem.cs

```diff
@@ -58,18 +60,44 @@ internal override void OnEntityComponentRemoved(EntityComponent component)
 
         internal override void OnSwapChainChanged()
         {
+            if (Game.Scene == null)
+                return;
+
+            var viewport = Game.Scene.Camera.Viewport;
+            var size = new Size(viewport.Width, viewport.Height);
+
             foreach (var guiComponent in _guiComponents)
             {
-                //CreateSizeDependentResources(guiComponent);
+                guiComponent.Layout(Game.GraphicsDevice, size);
             }
         }
 
-        public override void Update(GameTime gameTime)
+        internal override void BuildRenderList(RenderList renderList)
         {
-          
-            base.Update(gameTime);
+            foreach (var component in _guiComponents)
+            {
+                renderList.Gui.AddRenderItemDraw(
+                    component.Material,
+                    component.VertexBuffer,
+                    null,
+                    CullFlags.AlwaysVisible,
+                    null,
+                    default,
+                    0,
+                    6);
+            }
+
+            base.BuildRenderList(renderList);
         }
 
-       
+        public override void Update(GameTime gameTime)
+        {
+            foreach(var component in _guiComponents)
```

Comment:
I think you still have non-standard spacing settings - the default VS spacing settings have a space after keywords like `foreach`.